src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc

```diff
@@ -101,6 +101,20 @@ In the command above, `eclipse-che` is the Che namespace.
 
 Che server, Keycloak and PostgreSQL pods, and workspace pods use Persistent Volume Claims (PVCs), which are bound to the physical Persistent Volumes (PVs) with https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[ReadWriteOnce access mode]. When the deployment YAML files run, they define the Che PVCs. You can configure link:#storage-strategies-for-che-workspaces[workspace PVC] access mode and claim size with Che deployment environment variables.
 
+[IMPORTANT]
+====
+To run multiple workspaces, the follwoing two conditions must be met:
```

Comment:
```suggestion
To run multiple workspaces, the following two conditions must be met:
```
This is unclear; this suggests that running multiple workspaces requires the common PVC strategy and RWO volumes, when in actuality those conditions *prevent* running multiple workspaces.

Reply:
Thanks for pointing this out, @amisevsk . I have fixed it in https://github.com/eclipse/che-docs/pull/812/commits/c6c27e92ea7ca60c5692ec6ba0cf2765392223a0.